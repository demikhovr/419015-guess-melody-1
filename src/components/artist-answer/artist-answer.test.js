import React from 'react';
import renderer from 'react-test-renderer';
import ArtistAnswer from './artist-answer.jsx';

const mock = {
  id: 1,
  picture: ``,
  artist: `correct-artist`
};

it(`ArtistAnswer correctly renders`, () => {
  const {
    id,
    picture,
    artist,
  } = mock;
  const clickHandler = jest.fn();
  const tree = renderer
    .create(<ArtistAnswer
      id={id}
      picture={picture}
      artist={artist}
      onAnswer={clickHandler}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
