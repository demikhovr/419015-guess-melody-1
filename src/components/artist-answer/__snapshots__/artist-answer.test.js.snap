// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistAnswer correctly renders 1`] = `
<div
  className="artist"
>
  <input
    className="artist__input visually-hidden"
    id="answer-1"
    name="answer"
    onClick={[Function]}
    type="radio"
    value="answer-1"
  />
  <label
    className="artist__name"
    htmlFor="answer-1"
  >
    <img
      alt="correct-artist"
      className="artist__picture"
      src=""
    />
    correct-artist
  </label>
</div>
`;
